.jenkins/pytorch/build.sh

```diff
@@ -65,10 +65,10 @@ fi
 if [[ "${BUILD_ENVIRONMENT}" == *-android* ]]; then
   export ANDROID_NDK=/opt/ndk
   build_args=()
-  build_args+=("-DBUILD_BINARY=ON")
-  build_args+=("-DBUILD_TEST=ON")
-  build_args+=("-DUSE_OBSERVERS=ON")
-  build_args+=("-DUSE_ZSTD=ON")
+  build_args+=("-DBUILD_TORCH_MOBILE=ON")
+  build_args+=("-DCMAKE_PREFIX_PATH=$(python -c 'from distutils.sysconfig import get_python_lib; print(get_python_lib())')")
+  build_args+=("-DPYTHON_EXECUTABLE=$(python -c 'import sys; print(sys.executable)')")
+  build_args+=("-DPYTHON_INCLUDE_DIR=$(python -c 'from distutils.sysconfig import get_python_inc; print(get_python_inc())')")
```

Comment:
Why is this needed? Does the existing python logic in https://github.com/pytorch/pytorch/blob/master/cmake/Dependencies.cmake#L589 not work?

EDIT: Okay I see that BUILD_PYTHON is turned OFF below so that logic never runs. Why is python needed for the android build in the first place then?

Reply:
@pjh5 I think PYTHON_EXECUTABLE is needed by aten codegen script; PYTHON_INCLUDE_DIR might not be necessary, let me test it.